packages/grafana-ui/src/components/Tabs/TabBar.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { FC, ReactElement } from 'react';
+import { cx } from 'emotion';
+import { useTheme } from '../../themes';
+import { getTabsStyle } from './styles';
+import { TabProps } from './Tab';
+
+export interface Props {
+  /** Children should be a single <Tab /> or an array of <Tab /> */
+  children: ReactElement<TabProps> | Array<ReactElement<TabProps>>;
+  /** Provide a className if you wish to change the layout of the TabBar */
```

Comment:
Do we use this prop anywhere now? If not I think we should skip the className for now and implement the layout/orientation of Tabs in a separate PR